integration-tests/src/test/resources/performance-features/create-bridges-at-once.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Create Bridges performance tests
+
+  Background:
+    Given authenticate against Manager
+
+  Scenario Outline: Create many Bridges at once
+    When Create benchmark on Hyperfoil "hf-controller" instance with content:
+      """text/vnd.yaml
+      name: rhose-create-bridges
+      http:
```

Comment:
Shouldn't we define the number of agents as we did for the other perf scenario like

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/performance-features/cloud-events-performance.feature#L28-L30